src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
```

Comment:
you can just add fonts in `html` tag, there is no need to add them in each block